jekyll/_cci2/orb-intro.md

```diff
@@ -19,6 +19,32 @@ CircleCI orbs are open-source, shareable packages of parameterizable [reusable c
 
 Published orbs can be found on our [Orb Registry](https://circleci.com/developer/orbs), or you can [author your own orb]({{site.baseurl}}/2.0/orb-author-intro/).
 
+## Private orbs vs. public orbs
+
+There are two different types of orbs you can use in your configuration, depending on how you want to publish your orbs. If you prefer to publish your orb internally, and not to the [CircleCI Orb Registry](https://circleci.com/developer/orbs), you will want to use a Private Orb. However, if you want to publish your orb to the [CircleCI Orb Registry](https://circleci.com/developer/orbs), use a Public Orb. Descriptions of each type of orb is described in the sections below.
+
+### Private orbs
+
+**Note:** Private orbs are currently only available if you are on the https://circleci.com/pricing[Scale Plan]. Please reach out to your sales representative for information on how to sign up for the Scale Plan.
+
+Orbs are reusable chunks of configuration published to the [CircleCI Orb Registry](https://circleci.com/developer/orbs). Traditionally, these orbs are authored by individuals and then published to the entire world. But what should you do if you want to author an orb, but do not want the orb published outside of your organization? Using a private orb enables you to author an orb while ensuring the following:
```

Comment:
```suggestion
Using a private orb enables you to author an orb while ensuring the following:
```
The beginning of the document already states what is being stated here, I would remove it.